newIDE/app/src/UI/Theme/i18n.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import domain from './domain';
+
+const directionSpec = {
+  get direction() {
+    return domain['language-ltr'] ? 'unset' : 'rtl';
+  },
+};
+
+export const text = {
+  root: { root: directionSpec },
+  label: { label: directionSpec },
+};
+
+const orderSpec = {
+  get order() {
+    return domain['language-ltr'] ? 'unset' : (styles.order?.rtl ?? 100);
+  },
+};
```

Comment:
I'm afraid the accessor make the code hard to understand, as they are implicitly called, and they rely on a global variable.
In the future, we may have more than one instance of the editor running, with possible different theme or even language. To keep this flexibility, we must not rely on any global variable.

Instead, I suggest a different approach: 
* No more accessors at all in the theme.
* Keep the muiTheme as usual...
* ...but make a function that takes a muiTheme and return the rtl equivalent (we can call it `convertToRtl`)
* in `getTheme`, returns the normal `Theme` (i.e: `{gdevelopTheme, muiTheme}` if `isLtr(language)` is false, and return `{gdevelopTheme, muiTheme: convertToRtl(muiTheme)}` otherwise.